pkg/builtinchecks/yamls/latest-tag.yaml

```diff
@@ -1,9 +1,10 @@
 name: "latest-tag"
-description: "Indicates when a deployment-like object is running a container with a floating image tag, \"latest\""
-remediation: "Use a container image with a proper image tag, outside the set blocked tag regex \".*:(latest)$\"."
+description: "Indicates when a deployment-like object is running a container with an invalid container image"
+remediation: "Use a container image with a proper image tag satisfying the \"AllowList\" & \"BlockList\" regex patterns."
 scope:
   objectKinds:
     - DeploymentLike
 template: "latest-tag"
 params:
-  BlockList: [".*:(latest)$" ]
+  BlockList: [".*:(latest)$"]
+  AllowList: []
```

Comment:
We can just omit the `AllowList` if it's empty. Otherwise, it can be confusing.

Reply:
Removed the empty `AllowList`. 

Thanks for pointing.